app/viewModels/taskList/TaskStatus.scala

```diff
@@ -22,8 +22,9 @@ sealed trait TaskStatus {
   val messageKey: String
   val tag: String
   val jsonString: String
-  def isCompleted: Boolean = this == TaskStatus.Completed
-  def isError: Boolean     = this == TaskStatus.Error
+  val hiddenMessageKey: Option[String] = None
```

Comment:
Since this change is very specific to when there's an error, I'm not sure adding this value to the superclass really makes sense. It might be more prudent to be more explicit in the TaskList component
```
@task.status match {
  case CannotStartYet => { @task.name }
  case Error => {
    <a href="@{task.href(lrn)(config)}" aria-describedby="@task.id-status">@task.name
      <span class="govuk-visually-hidden" id="@task.id-hidden">@messages("taskStatus.error.hidden")</span>
    </a>
  }
  case _ => { <a href="@{task.href(lrn)(config)}" aria-describedby="@task.id-status">@task.name</a> }
```